apps/pdc-frontend/src/util/generateSurveyURL.ts

```diff
@@ -0,0 +1,58 @@
+export interface SurveyRunParams {
+  guid?: string;
+  view?: string;
+  apikey?: string;
+  istestresult?: string;
+  locale?: string;
+  segment?: string;
+  frontendPublicURL?: string;
+  surveyURL?: URL;
+}
+const frontendPublicURL = new URL(process.env.FRONTEND_PUBLIC_URL || 'http://localhost:3000');
+const surveyURL = new URL(process.env.SURVEY_RUN_URL ?? '');
+
+export const surveyParams = {
+  guid: process.env.SURVEY_RUN_GUID,
+  view: 'standard',
+  apikey: process.env.SURVEY_RUN_APIKEY,
+  istestresult: 'false',
+  locale: 'nl',
+  frontendPublicURL: frontendPublicURL.origin,
+  surveyURL,
+};
```

Comment:
Ik zou utility functions maken als "pure functies", dat betekent dat ze met dezelfde input altijd dezelfde output hebben. `process.env` is variabel, en dat maakt dat het geen pure functie is. Ik zou iets maken dat zo kan werken:

```js
const surveyHref = generateSurveyURL(getSurveryParams(process.env));
```